src/main/java/org/jfrog/hudson/generic/ArtifactoryGenericConfigurator.java

```diff
@@ -379,6 +412,23 @@ public boolean tearDown(AbstractBuild build, BuildListener listener) {
         };
     }
 
+    private PublisherContext.Builder getBuilder() {
```

Comment:
This method seems redundant.

Reply:
Correct. I've also removed the import reference for it.